packages/utils/src/lib/reports/__snapshots__/report-diff-added.md

```diff
@@ -0,0 +1,28 @@
+# Code PushUp
+
+😟 Code PushUp report has **regressed** – compared target commit `0123456` with source commit `abcdef0`.
+
+## 🏷️ Categories
+
+|🏷️ Category|⭐ Current score|⭐ Previous score|🗠 Score change|
+|:--|:--:|:--:|:--:|
+|Bug prevention|🟡 **63**|🟡 68|![🠋 −5](https://img.shields.io/badge/%F0%9F%A0%8B%20%E2%88%925-red)|
+|Performance|🟢 **94**|n/a|n/a|
```

Comment:
Wondering about the n/a... if we could add **(new)** somewhere instead or put `n/a` in code format maybe...can't quite put my finger on it. WDYT?

Reply:
Are any of these better?

![image](https://github.com/code-pushup/cli/assets/34691111/b6f374b6-d666-45ac-af78-f80f7693a91e)

Or with an asterisk:

![image](https://github.com/code-pushup/cli/assets/34691111/1c6aae1a-4586-491a-92d6-2132431709c7)

Reply:
`n/a` in code wrapper and * without ()

Reply:
After looking at this, I am a fan of the italics, be it with or without an asterisk. When it comes to the asterisk, I prefer it with the parentheses as it is done e.g. in our generated zod schema documentation.

Reply:
Went with italics and asterisk in parentheses.